Firmware/Marlin_main.cpp

```diff
@@ -3618,6 +3609,19 @@ void process_commands()
   }
 
 #ifdef TMC2130
+  /*!
```

Comment:
I prefer to have the documentation above and not splittend.
During the documentation i found quite a lot of these splits and people forgot to update these over time.
Having it together make more sense to me. Can you please change this back.

Reply:
Did the changes